proto/cosmos/base/snapshots/v1beta1/snapshot.proto

```diff
@@ -66,19 +64,4 @@ message SnapshotExtensionMeta {
 // Since: cosmos-sdk 0.46
 message SnapshotExtensionPayload {
   bytes payload = 1;
-}
-
-// SnapshotKVItem is an exported Key/Value Pair
```

Comment:
> but because it was never used

We did release 0.46 with this proto message, right? In this case I think we need to keep it unfortunately (also on main too)

Reply:
Ah! :( Should we deprecate it with a comment that says it is not being used anywhere?

Reply:
Yes, that's what we did for other unused fields: add a comment and the proto `deprecated=true` annotation

Reply:
Ok! I thought it only mattered for fields that were actually used. I will update then, thanks!

Reply:
id just delete it tbh. it snot used by anything and if anyone is using it then they are using an internal api externally

Reply:
The problem is that once it's exposed in proto files, then it's really not an internal API anymore, and even JS clients can consume them.

Seems to me that there's a risk of breaking backwards-compatibility if we remove, whereas the alternative of deprecation is safe and easy.

Reply:
sure, but if anyone is using this then they are doing something wrong so I still think deleting is simpler than legacy/dead code sitting round